spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/datastore/GcpDatastoreAutoConfiguration.java

```diff
@@ -77,7 +77,14 @@
 				: projectIdProvider.getProjectId();
 		this.namespace = gcpDatastoreProperties.getNamespace();
 
-		if (gcpDatastoreProperties.getEmulatorHost() == null) {
+		String hostToConnect = gcpDatastoreProperties.getHost();
+		if (gcpDatastoreProperties.getEmulator().isEnabled()) {
+			if (hostToConnect == null) {
```

Comment:
If the emulator is enabled, why would anyone still want to connect to something else? I think we can just say that `spring.cloud.gcp.datastore.host` is ignored if the emulator is enabled and remove this check.

Reply:
Ok